### ISSUE TITLE
Wrong foreground browser window

### ISSUE THREAD

**Issue by tapapax** [tapapax/firefox-fts#21]
- Open two browser windows above each other
- Open FTS popup in window 1 and select any tab from window 2
--> Focus is gained by correct win and tab, but foreground window is 1 (should be 2)

Seems like window.close() on popup is making it's owner window foreground.

**Comment by tapapax** [comment]
@dmitry-timofeev - popup-related issue:
Could you check if https://github.com/tapapax/firefox-fts/commit/1f4f46864066536234c716bff1b6283a4236ff42 is working for you please?

**Comment by dmitry-timofeev** [comment]
Strange, couldn’t reproduce the issue with 2.3.0. 1f4f46864066536234c716bff1b6283a4236ff42 works fine.

However, I bumped into another issue (it might be a Firefox one): if the extension button is hidden because the window size is too narrow, the browser action is not delivered on `Ctrl+Space`. If the extension button is always hidden inside "More tools…" menu, the browser action *is* delivered (but the user sees an intermediate "More tools…" menu for a short period of time).

**Comment by dmitry-timofeev** [comment]
An error message when the extension button is hidden:
```
Cannot show a subview in a closed panel. PanelMultiView.jsm:607
_showSubView
resource:///modules/PanelMultiView.jsm:607:22
InterpretGeneratorResume self-hosted:1266:8 next self-hosted:1221:9 showSubView
resource:///modules/PanelMultiView.jsm:596:5
value
resource:///modules/PanelMultiView.jsm:410:42
showSubView
chrome://browser/content/customizableui/panelUI.js:375:7
next self-hosted:1221:9 handleWidgetCommand
resource:///modules/CustomizableUI.jsm:1651:7
handleWidgetCommand self-hosted:1085:17 triggerAction
chrome://browser/content/parent/ext-browserAction.js:270:7
InterpretGeneratorResume self-hosted:1266:8 next self-hosted:1221:9 buildKey/<
chrome://browser/content/parent/ext-commands.js:215:9
```

The diff in which this check was added (quite recent): https://searchfox.org/mozilla-central/diff/91212fa6b1ab5b0de9a8ffee7316c55e74b50d0a/browser/components/customizableui/PanelMultiView.jsm#611

**Comment by tapapax** [comment]
Fixed in 1f4f468
